src/gui/mzroll/isotopeplot.cpp

```diff
@@ -1,27 +1,24 @@
 #include "isotopeplot.h"
 
-// import most common Eigen types
-//USING_PART_OF_NAMESPACE_EIGEN
 using namespace Eigen;
 
+IsotopePlot::IsotopePlot(QCustomPlot* customPlot, float width, float height, float stackingZValue, 
```

Comment:
@IpankajI Do we need to pass customPlot here? Can't we make customPlot as private member of IsotopePlot class?

Reply:
@IpankajI Also, why do we need to pass width and height even?

Reply:
customPlot is used to hold bars and manage these, but this needs to be placed somewhere else in DockWidget which in turn has to be placed in a GUI element, not as independent entity. So it has to be public and it's height and width has to be specified based on holding GUI element.

Reply:
@sahil21 done!